diambra/arena/make_env.py

```diff
@@ -1,12 +1,13 @@
 import os
 import logging
-from dacite import from_dict
-from .arena_gym import DiambraGymHardcore1P, DiambraGym1P, DiambraGymHardcore2P, DiambraGym2P
-from .wrappers.arena_wrappers import env_wrapping
-from .env_settings import EnvironmentSettings1P, EnvironmentSettings2P, WrappersSettings, RecordingSettings
+from dacite import from_dict, Config
+from diambra.arena.arena_gym import DiambraGym1P, DiambraGym2P
+from diambra.arena.wrappers.arena_wrappers import env_wrapping
+from diambra.arena.env_settings import EnvironmentSettings1P, EnvironmentSettings2P, WrappersSettings, RecordingSettings
+from diambra.arena.wrappers.episode_recording import EpisodeRecorder
 
-def make(game_id, env_settings={}, wrappers_settings={},
-         traj_rec_settings={}, seed=None, rank=0, log_level=logging.INFO):
+def make(game_id, env_settings:dict={}, wrappers_settings:dict={}, episode_recording_settings:dict={},
```

Comment:
Can't we use the EnvSetting class here as a type hint instead of a generic dict?

Reply:
This is a very clear point and I was sure it was going to come out as I know your expertise ;)

I evaluated multiple options here. There was one that can be even more interesting, meaning directly exposing the protobuffer EnvSettings class, it can be directly populated by a json/dictionary so the user could just provide a properly defined dict and in a oneliner we could load that dict into the protobuffer class. 
```
# Use ParseDict to populate the Company message
your_proto_file_pb2.ParseDict(data_dict, company_message)
```
and it would mean to completely remove "translation" from a datastructure to another.

But. The main reason why I ask for _simple_ dictionaries here, is that this is the main standard in the field. People are used to them, and I believe they are easier to understand, not requiring proper nesting, that is instead needed by protobuffer.

This is also the reason why I would avoid exposing the EnvSettings dataclass, to keep dictionaries. If we want to go no-standard, I would rather select the json-dictionary to be directly loaded in the PB message.

But I would also need to restructure how we handle the random values at restart, which is currently taken care of by the intermediate class.

Let me know your thoughts about this point, I am really interested in knowing your opinion.

Reply:
What about making `make` take EnvSettings and users can either:
- use diambra.engine.models's EnvSetting's ParseDict
- use a diambra.arena method that parses a 'flat' generic dict into EnvSetting which can be called like this:
```
env = make(env_settings=settings_from_dict(env_settings)..)
```

We could also consider:
- having different make functions for using the flat dict or the pb one
- having env_settings_dict as additional argument and require user to provide either env_settings_dict or env_settings with the pb typed on
- use type assertion in make to figure out if a flat dict or EnvSettings dataclass got passed in

But I prefer explicit over implict so exposing a `settings_from_dict`, avoid hidden logic in make and removing any "translation" seems like the best option to me.

Reply:
Just pushed a commit for this: https://github.com/diambra/arena/commit/32120db4bac86f804493830f0ff41b61f460f7b4#diff-156151d5440a63225c4f5517120a361f47d54f2299c70f3d3ac6e5dc2df95c0a